pkg/middlewares/aws_eip_association_filter.go

```diff
@@ -0,0 +1,47 @@
+package middlewares
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+)
+
+/**
+  Fetching eip association from remote return every association but some of them are embedded in eip.
+  This middleware will check for every eip_association that here is no corresponding association_id inside eip.
+*/
+
+type EipAssociationFilter struct {
+}
+
+func NewEipAssociationFilter() EipAssociationFilter {
+	return EipAssociationFilter{}
+}
+
+func (m EipAssociationFilter) Execute(remoteResources, resourcesFromState *[]resource.Resource) error {
+	var newResources []resource.Resource
+	for _, res := range *remoteResources {
+		if res.TerraformType() != aws.AwsEipAssociationResourceType {
+			newResources = append(newResources, res)
+			continue
+		}
+		if m.matchingEipAssociationId(res, resourcesFromState) {
```

Comment:
I'm wondering if this isn't causing issues when in your state you declare an eip_assoc resource ? Have you tested this behavior ? Maybe it show resource as missing since you filter it from remote (in some case maybe we can have `association_id` beside a dedicated eip assoc resource, after a TF refresh or so ?)

Reply:
I tried with ou acceptancy test on eip and it seemed to work I don't know about a refresh I'll test that

Reply:
OK I tested with a refresh without any issue